### ISSUE TITLE
Persist parser and parser settings in URL

### ISSUE THREAD

**Issue by fkling** [fkling/esprima_ast_explorer#55]
To make sharing code snippets easier, it would be convenient to store the selected parser and parser settings in the URL.

However, I don't really like to update the current URI with that information, since it will look ugly and mess with the browser history. Instead, we could add a "share" button that shows a copyable version of the URL, just like jsfiddle does.

I'm already working on this.

**Comment by RReverser** [comment]
Half-done with introduction of saving parser?

**Comment by fkling** [comment]
Yeah... different from what I intended to do, but sufficient enough :)
